cpp/dolfinx/fem/DirichletBC.h

```diff
@@ -131,6 +131,62 @@ template <typename T>
 class DirichletBC
 {
 private:
+  /// @brief Mark cells with bc dofs
+  ///
+  /// Returns an array of size number of local cells, where 1 indicates that the
+  /// cell has a dof from the incoming list in the dofmap
+  /// @param[in] V The function space
+  /// @param[in] dofs The degrees of freedom (local indices, unrolled),
```

Comment:
Which degrees of freedom? Doc string could be clearer.

Reply:
I've tried to clarify this now by renaming the convenience function and giving it proper documentation.